index.js

```diff
@@ -95,6 +95,19 @@ module.exports.options = {
     watch: {
         default: false,
         runtimeParameter: 'watch'
+    },
+    serializers: {
+        default: {},
+        marks: {
+            default: {
+                annotations: {
+                    default: [],
+                },
+            }
+        },
+        types: {
+            default: {},
+        },
```

Comment:
I think the `parsePluginOptions` function ([defined here](https://github.com/stackbit/sourcebit/blob/1dde98badc428a87d706bcee0e4c41ffe4eed29e/lib/sourcebit.js#L127-L151)) doesn't recursively merge default values of nested properties. It only looks at the root `default` value of every root option.

So I guess the right way would be specifying a single `default` object with possible properties:
```
serializers: {
  default: {
    marks: {},
    types: {}
  }
}
```

You can also set the `serializers` options to be an empty object, like the `dataset` and `projectId` options. This is fine because now the code access the `serializers` nested properties in a safe way, i.e.: `(options.serializers && options.serializers.types)`:

```
serializers: {}
```